source/module_elecstate/occupy.cpp

```diff
@@ -131,8 +131,12 @@ void Occupy::iweights(const int nks,
                       const std::vector<int>& isk)
 {
     assert(is < 2);
+    double degspin = 2.0;
+    if (GlobalV::NSPIN == 4)
+        degspin = 1.0;
+    if (is != -1)
```

Comment:
Sorry, I think I didn't understand the difference between new code and old code.
The judgement of "is != -1" is not same as "GlobalV::NSPIN == 2"?

Reply:
When use TWO_EFERMI, is = 0 or 1.  When not use TWO_EFERMI， is = -1。
To be honest，I am not familiar with TWO_EFERMI and I just copy the code of QE.